desktop-widgets/mainwindow.cpp

```diff
@@ -636,6 +637,7 @@ bool MainWindow::plannerStateClean()
 
 void MainWindow::planCanceled()
 {
+	profile->view->setProfileState(plannerWidgets->getDive(), profile->dc);
```

Comment:
Is that still necessary? The `showProfile()` call in the next line will either show the empty profile or the current dive, and thus reset the profile's status.